database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+Drop the existing PIMS_CONTACT_MGR_VW view and recreate to reference the 
+'MAILING' code value for ADDRESS_USAGE_TYPE_CODE rather than the deprecated 
+'MAILADDR' code value.
+*******************************************************************************/
+
+DROP VIEW IF EXISTS PIMS_CONTACT_MGR_VW;
+GO
+
+CREATE VIEW PIMS_CONTACT_MGR_VW AS
+  SELECT CONCAT('P', PER.PERSON_ID) AS ID
+       , PER.PERSON_ID
+       , NULL                   AS ORGANIZATION_ID
+       , PER.IS_DISABLED
+       , TRIM(CONCAT_WS(' ', PER.FIRST_NAME, PER.MIDDLE_NAMES, PER.SURNAME)) AS SUMMARY
+       , PER.SURNAME
+       , PER.FIRST_NAME
```

Comment:
Not sure this change belongs here? the changelog below only lists changes to the PIMS_COUNTRY ordering...

Reply:
Also the file name seems wrong - `..._16.01.sql` doesn't seem to indicate it should be applied in 16.03

Reply:
^^ @dfilteau ?

Reply:
I think the 16.01 PR has 16.03 added onto it.

Reply:
Any change you could delete this file from the PR? It is already under folder **S16_02** Alter Up so no sense in having it for folder **S16_03**

Reply:
^ @dfilteau

Reply:
Yup.  Can do.